src/app/Console/Commands/Install.php

```diff
@@ -26,45 +29,182 @@ class Install extends Command
      */
     protected $description = 'Install Backpack requirements on dev, publish files and create uploads directory.';
 
+    /**
+     * Addons variable.
+     *
+     * @var array
+     */
+    protected $addons = [
+        Addons\RequirePro::class,
+        Addons\RequireDevTools::class,
+        Addons\RequireEditableColumns::class,
+    ];
+
     /**
      * Execute the console command.
      *
      * @return mixed Command-line output
      */
     public function handle()
     {
-        $this->progressBar = $this->output->createProgressBar(5);
-        $this->progressBar->minSecondsBetweenRedraws(0);
-        $this->progressBar->maxSecondsBetweenRedraws(120);
-        $this->progressBar->setRedrawFrequency(1);
-
-        $this->progressBar->start();
-
-        $this->info(' Backpack installation started. Please wait...');
-        $this->progressBar->advance();
+        $this->infoBlock('Backpack installation started.');
 
-        $this->line(' Publishing configs, views, js and css files');
+        // Publish files
+        $this->progressBlock('Publishing configs, views, js and css files');
         $this->executeArtisanProcess('vendor:publish', [
-            '--provider' => 'Backpack\CRUD\BackpackServiceProvider',
+            '--provider' => BackpackServiceProvider::class,
             '--tag' => 'minimum',
         ]);
+        $this->closeProgressBlock();
 
-        $this->line(" Creating users table (using Laravel's default migration)");
+        // Create users table
+        $this->progressBlock('Creating users table');
         $this->executeArtisanProcess('migrate', $this->option('no-interaction') ? ['--no-interaction' => true] : []);
+        $this->closeProgressBlock();
 
-        $this->line(" Creating App\Http\Middleware\CheckIfAdmin.php");
+        // Create CheckIfAdmin middleware
+        $this->progressBlock('Creating CheckIfAdmin middleware');
         $this->executeArtisanProcess('backpack:publish-middleware');
+        $this->closeProgressBlock();
+
+        // Install Backpack Generators
+        $this->progressBlock('Installing Backpack Generators');
+        // $process = new Process(['composer', 'require', '--dev', 'backpack/generators']);
+        // $process->setTimeout(300);
+        // $process->run();
+        $this->closeProgressBlock();
+
+        // Create users
+        $this->createUsers();
+
+        // Addons
+        $this->installAddons();
+
+        // Done
+        $url = Str::of(config('app.url'))->finish('/')->append('admin/');
+        $this->infoBlock('Backpack installation complete.', 'done');
```

Comment:
I see you've changed the blue INFO label to DONE here - didn't know we can do that. In this case, why don't we move all `infoBlock()` calls to the `php artisan backpack:install` command, and name them `Step 1`, `Step 2`, `Step 3`? That way:
- if you run that command individually, you won't have info blocks (you don't really need them, you know what that command is doing already, it's in the name);
- if you run that command as part of `php artisan backpack:install`, it'll read as a smooth, step-by-step process:
![CleanShot 2022-08-01 at 08 21 45](https://user-images.githubusercontent.com/1032474/182077909-07979f47-44c3-4044-abbb-bdc6ba62072a.png)